lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/TaskManager.cs

```diff
@@ -347,12 +370,36 @@ internal void RecordRunningTaskDuringSystemFailure(IRunningTask runningTask, str
 
         /// <summary>
         /// Gets error type based on the exception type in IFailedTask 
+        /// For unknown exceptions or exceptions that doesn't belong to defined IMRU task exceptions
+        /// treat then as application error.
         /// </summary>
         /// <param name="failedTask"></param>
         /// <returns></returns>
         private TaskStateEvent GetTaskErrorEventByExceptionType(IFailedTask failedTask)
```

Comment:
Comments and what this function do do not match? It doe snot return errortypr right but general TaskStateEvent.